tox.ini

```diff
@@ -0,0 +1,22 @@
+[flake8]
+max-line-length = 88
+
+[tox]
+envlist = py36, py37, py38, flake8, mypy, black
+skip_missing_interpreters = True
+
+[testenv]
+deps = pytest
+commands = pytest
+
+[testenv:flake8]
+deps = flake8
+commands = flake8 tests/ potodo/
+
+[testenv:black]
+deps = black
+commands = black --check --diff tests/ potodo/
+
+[testenv:mypy]
+deps = mypy
+commands = mypy --ignore-missing-imports potodo/
```

Comment:
Can you add a todo here to add #strict?

Reply:
There's already an issue about it (https://github.com/Seluj78/Potodo/issues/30) I don't feel committing it as a comment.